### ISSUE TITLE
Camera layout is incostistent

### ISSUE THREAD

**Issue by Turbolqk** [whatawurst/android_device_sony_yoshino-common#104]
Aperture shows the camera preview in a mini-windows. Switching 4:3 to 16:9 doesn't affect the screen, but affects the photos taken.
Might be upstream and/or resolution problem.

**Comment by derfelot** [comment]
Yes, pretty sure this is either an upstream issue, or an incompatibility with our camera stack from Sony stock. I'll see if there is anything that can be done about it.

**Comment by Turbolqk** [comment]
Just to note, GrapheneOS' camera seems to work right. Might be of some help?
https://github.com/GrapheneOS/Camera
